### PR TITLE
Empty ContainsAny should return false

### PR DESCRIPTION
**Pull request by janmarques**
http://issues.hibernatingrhinos.com/issue/RDBC-76

### CODE REVIEW

**Comment by wiezykrwi** [comment]
Issue: http://issues.hibernatingrhinos.com/issue/RDBC-76

File: src/Raven.Client/Documents/Session/Tokens/FalseToken.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    {
+        private FalseToken()
+        {
+        }
+
+        public static readonly FalseToken Instance = new FalseToken();
+
+        public override void WriteTo(StringBuilder writer)
+        {
+            writer
```
Comment (ayende) [inline src/Raven.Client/Documents/Session/Tokens/FalseToken.cs:15]: There is actually no support for `false` in RQL

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.Client/Documents/Session/Tokens/FalseToken.cs (hunk 1 above)
Comment (jeroenlorrentop) [reply src/Raven.Client/Documents/Session/Tokens/FalseToken.cs:15]: Other ways to fix this? Or throw an exception. Having the opposite is a bit ackward, isn't it?

Comment (ayende) [reply src/Raven.Client/Documents/Session/Tokens/FalseToken.cs:15]: I assigned the issue, we'll look int this.

Comment (janmarques) [reply src/Raven.Client/Documents/Session/Tokens/FalseToken.cs:15]: Thanks

**Pull request closed without merging**